src/plugins/discover/public/components/doc_table/doc_table_wrapper.tsx

```diff
@@ -196,9 +196,7 @@ export const DocTableWrapper = forwardRef(
       (rowsToRender: DocTableRow[]) => {
         return rowsToRender.map((current) => (
           <TableRow
-            key={`${current._index}${current._type ?? ''}${current._id}${current._score}${
-              current._version
-            }${current._routing}`}
+            key={`${current._index}${current._id}${current._score}${current._version}${current._routing}`}
```

Comment:
I don't think it should affect the unit test, but they fail. @kertal would you mind taking a look, please?

Reply:
@mshustov I could push a fix for this if you don't mind?